gcp-observability/src/main/java/io/grpc/gcp/observability/GcpObservability.java

```diff
@@ -45,23 +69,42 @@ public static synchronized GcpObservability grpcInit() throws IOException {
     if (instance == null) {
       GlobalLoggingTags globalLoggingTags = new GlobalLoggingTags();
       ObservabilityConfigImpl observabilityConfig = ObservabilityConfigImpl.getInstance();
-      Sink sink = new GcpLogSink(observabilityConfig.getDestinationProjectId(),
-          globalLoggingTags.getLocationTags(), globalLoggingTags.getCustomTags(),
-          observabilityConfig.getFlushMessageCount());
+      Sink sink =
+          new GcpLogSink(
+              observabilityConfig.getDestinationProjectId(),
+              globalLoggingTags.getLocationTags(),
+              globalLoggingTags.getCustomTags(),
+              observabilityConfig.getFlushMessageCount());
+      // TODO(dnvindhya): Cleanup code for LoggingChannelProvider and LoggingServerProvider
+      // once ChannelBuilder and ServerBuilder are used
```

Comment:
Since the other PR (https://github.com/grpc/grpc-java/pull/9312) has been merged, we can clean up the Logging*Providers as part of this PR now provided you rebase and are able to test/verify that all the use cases work without the Logging*Providers. We can discuss

Reply:
As discussed offline, I will do this in a separate PR.